### ISSUE TITLE
Laravel 5.4 Compatibility

### ISSUE THREAD

**Issue by sschlein** [mpociot/teamwork#72]
The `registerCommands` function of the TeamworkServiceProvider throws an error because it uses the removed share method. I wasn't able to fix it, otherwise, I'd have provided a Pull Request.

It works if I remove the function body locally but this also removes the function.

```
    /**
     * Register scaffolding command
     */
    protected function registerCommands()
    {
        //$this->app['make.teamwork'] = $this->app->share(function () {
        //    return new MakeTeamwork();
        //});

        //$this->commands([
        //    'make.teamwork'
        //]);
    }
```

**Comment by sschlein** [comment]
Finally, I've been able to fix it. Took at look how Laravel Dusk does it 😃 

https://github.com/mpociot/teamwork/pull/73

**Comment by mpociot** [comment]
I just released 5.0.0 which gives Laravel 5.4 support :)
